# filepath: params.yaml
drive:
  ros__parameters:
    linear_x_coeff: 1.0
    linear_x_offset: 0.0
    linear_x_min: -1000.0
    linear_x_max: 0.5

    linear_y_coeff: 1.0
    linear_y_offset: 0.0
    linear_y_min: -1000.0
    linear_y_max: 1000.0

    linear_z_coeff: 1.0
    linear_z_offset: 0.0
    linear_z_min: -1000.0
    linear_z_max: 1000.0

    angular_x_coeff: 1.0
    angular_x_offset: 0.0
    angular_x_min: -1000.0
    angular_x_max: 1000.0

    angular_y_coeff: 1.0
    angular_y_offset: 0.0
    angular_y_min: -1000.0
    angular_y_max: 1000.0

    angular_z_coeff: 0.2
    angular_z_offset: 0.0
    angular_z_min: -0.26
    angular_z_max: 0.6

    controller_type: "bicycle"
    output_frame_id: "drive"

aeb:
  ros__parameters:
<<<<<<< HEAD
    ttc_threshold: 2.0
=======
    ttc_threshold: 5.0
>>>>>>> e3af4822991780a21cd8b2e6cc98a060a4f1a198
    aeb_aktiv: 1
    controller_type: "bicycle"